utils/urlUtils.js

```diff
@@ -0,0 +1,26 @@
+function getUrlDetails(url) {
+    let urlDetails = {
+        project : "",
+        type : "",
+        id : ""
+    }
+
+    if(url.lastIndexOf('pull') === -1)
+        urlDetails.type = "issue";
+    else if(url.lastIndexOf('issues') > url.lastIndexOf('pull'))
+        urlDetails.type = "issue";
+    else
+        urlDetails.type = "pull";
+
```

Comment:
What if the repo name consist of 'issues' or 'pull'?

Reply:
if repo name contains pull and link is issue condition 2 will work.
if repo name contains issue and link is issue condition 1 will work.
otherwise its pull else case